cellpack/autopack/ingredient/grow.py

```diff
@@ -2574,9 +2581,9 @@ def add_rb_multi_sphere(self):
             )  #
         return inodenp
 
-    def jitter_place(
+    def grow_place(
```

Comment:
I don't think this should be renamed. It makes sense for different classes to have their own implementation of the same function. So if placetype == 'jitter' whatever implementation of jitter that ingredient has, it will use. That's the whole point of putting it in it's own class